test/docker.sh

```diff
@@ -29,6 +29,6 @@ export IMAGE=makeomatic/alpine-node:$NODE_VER
 $COMPOSE -f $DC up -d
 
 # make sure that services are up
-sleep 5
+sleep 25
```

Comment:
While naive Cassandra images take around 30 seconds to start, our containers take only a few seconds. Optimizations include:

Disabling vnodes. We don't use them at Spotify, and Cassandra starts much faster without them (~10 sec).
Disabling something called "waiting for gossip to settle down" because there is no gossip in a one-node cluster (another ~10 sec).